src/ex1/script.js

```diff
@@ -0,0 +1,100 @@
+const taskInput = document.querySelector(" #taskInput ");
+const addBtn = document.querySelector(" #addBtn ");
+const clearAllBtn = document.querySelector(" #clearAllBtn ");
+const sortAllBtn = document.querySelector(" #sortAllBtn ");
+const taskList = document.querySelector(" #taskList ");
```

Comment:
having all the elements outside here requires all the dom elements exist when this file is loaded, which might not be the case in complex web apps that dynamically change the dom depending on the view